packages/2d/src/components/QuadBezier.ts

```diff
@@ -0,0 +1,98 @@
+import {SignalValue} from '@motion-canvas/core/lib/signals';
+import {PossibleVector2, Vector2Signal} from '@motion-canvas/core/lib/types';
+import {QuadBezierSegment} from '../curves';
+import {PolynomialSegment} from '../curves/PolynomialSegment';
+import {computed, vector2Signal} from '../decorators';
+import {lineTo, moveTo, quadraticCurveTo} from '../utils';
+import {Bezier, BezierOverlayInfo} from './Bezier';
+import {CurveProps} from './Curve';
+
+export interface QuadBezierProps extends CurveProps {
+  p0: SignalValue<PossibleVector2>;
+  p0X: SignalValue<number>;
+  p0Y: SignalValue<number>;
+
+  p1: SignalValue<PossibleVector2>;
+  p1X: SignalValue<number>;
+  p1Y: SignalValue<number>;
+
+  p2: SignalValue<PossibleVector2>;
+  p2X: SignalValue<number>;
+  p2Y: SignalValue<number>;
+}
+
+/**
+ * A node for drawing a quadratic Bézier curve.
+ *
+ * @example
+ * ```tsx editor
+ * export default makeScene2D(function* (view) {
+ *   const bezier = createRef<QuadBezier>();
+ *
+ *   view.add(
+ *     <QuadBezier
+ *       ref={bezier}
+ *       lineWidth={4}
+ *       stroke={'lightseagreen'}
+ *       p0={[-200, 0]}
+ *       p1={[0, -200]}
+ *       p2={[200, 0]}
+ *       end={0}
+ *     />
+ *   );
+ *
+ *   yield* bezier().end(1, 1);
+ *   yield* bezier().start(1, 1).to(0, 1);
+ * });
+ * ```
+ */
+export class QuadBezier extends Bezier {
+  /**
+   * The start point of the Bézier curve.
+   */
+  @vector2Signal('p0')
+  public declare readonly p0: Vector2Signal<this>;
+
+  /**
+   * The control point of the Bézier curve.
+   */
+  @vector2Signal('p1')
+  public declare readonly p1: Vector2Signal<this>;
+
+  /**
+   * The end point of the Bézier curve.
+   */
+  @vector2Signal('p2')
+  public declare readonly p2: Vector2Signal<this>;
+
+  public constructor(props: QuadBezierProps) {
+    super(props);
+  }
+
+  @computed()
+  protected segment(): PolynomialSegment {
+    return new QuadBezierSegment(this.p0(), this.p1(), this.p2());
+  }
+
+  @computed()
+  protected overlayInfo(matrix: DOMMatrix): BezierOverlayInfo {
```

Comment:
Although it would be an interesting feature, right now computed methods don't get updated when called with a different set of arguments. This means that the value returned by `overlayInfo` will not be updated when the `matrix` change.

Repro: select the node and zoom out the preview:
![image](https://user-images.githubusercontent.com/64662184/229556093-f01b5330-ebee-42a7-bf09-026938d9377c.png)